go-controller/pkg/ovn/ovn.go

```diff
@@ -629,6 +658,28 @@ func (oc *Controller) WatchNodes() error {
 				klog.Warningf(err.Error())
 				gatewaysFailed.Store(node.Name, true)
 			}
+
```

Comment:
I would say that you need to split this out into its own namespace handler so that you can isolate the egress firewall functionality, and address: https://github.com/ovn-org/ovn-kubernetes/pull/1432#issuecomment-656204140

Reply:
@alexanderConstantinescu it was determined that the egressFirewall would have such small footprint when no objects are present that it doesn't make sese to isolate egressFirewall using a cli flag